README.md

```diff
@@ -178,6 +178,26 @@ The "Delete Staging Deployment" action allows you to delete the deployment not r
           app-name: <app name>
 ```
 
+## Arguments
+> [!NOTE]
+> Some arguments are only applicable for certain settings of the `action` argument. The Action column below specifies the pertinent actions for each argument. Any argument listed as Required is only required for the pertinent Action(s).
+
+|Argument|Action&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|Description|
+|--- |--- |--- |
+|`action`<br/>Action|all|(Required) The action to be performed by this task.<br/>One of: `deploy`, `set-production`, `delete-staging-deployment`<br/>Default value: `deploy`|
+|`azure-subscription`<br/>Azure Subscription ID|all|(Required) The id of the azure subscription.|
```

Comment:
Suggest to add an extra column for `Required`, and remove the `(xxx)` in description.

Specifically for this one: `The Azure subscription ID for the target Azure Spring Cloud instance.`